book/website/community-handbook/enter-the-project.md

```diff
@@ -0,0 +1,89 @@
+(ch-enter-the-project)=
+
+# How to enter *The Turing Way*
+
+*The Turing Way* is an open source project that involves a diverse community of researchers in making data science reproducible, ethical and inclusive for everyone.
```

Comment:
* I think this introduction could be shorter and punchier. E.g. I do not think it is necessary to explain what the Turing Way in this chapter.
* I think there should be a sub-heading to indicate this is an introduction